app/javascript/spotlight/admin/croppable_modal.js

```diff
@@ -0,0 +1,99 @@
+import Crop from 'spotlight/admin/crop';
+
+export default class CroppableModal {
+
+  attachModalHandlers() {
+    // Attach handler for when modal first loads, to show the cropper
+    this.attachModalLoadBehavior();
+    // Attach handler for save by checking if clicking in the modal is on a save button
+    this.attachModalSaveHandler();
+  }
+
+  attachModalLoadBehavior() {
+    var context = this;
+    // Listen for event thrown when modal is displayed with content
+    document.addEventListener('show.blacklight.blacklight-modal', function(e) {      
+      var dataCropperDiv = $('#blacklight-modal [data-behavior="iiif-cropper"]');
+      
+      if(dataCropperDiv) {
+        var dataCropperKey = dataCropperDiv.data('cropper-key');
+        var itemIndex = dataCropperDiv.data('index-id');
+        new Crop(dataCropperDiv, false).render();
+      }
+    });
+  }
+
+  setRegionField(dataCropperKey, itemIndex) {
+    var regionField = $('#blacklight-modal input[name="select_image_region');
+    var itemElement = $('[data-cropper="' + dataCropperKey + '"]');
+    var thumbnailUrl = this.iiifInputField(itemIndex, 'thumbnail_image_url', itemElement).val();
+    var region = this.extractRegionField(thumbnailUrl);
+    regionField.val(region);
+    return regionField;
+  }
+  //When editing an existing/saved item, extract region values based on url
+  extractRegionField(iiifThumbnailUrl) {
+    if (iiifThumbnailUrl != null && iiifThumbnailUrl.length == 0) return null;
+
+    var regex = /\/[0-9]+,[0-9]+,[0-9]+,[0-9]+\//;
```

Comment:
This seems like a kind of messy way to extract the data. I think if you really need this data you can add it to the crop dom with one line of code here: https://github.com/projectblacklight/spotlight/blob/105986e9bfc48081d7a754398a9eb1a17cb5f787/app/javascript/spotlight/admin/crop.js#L177-L178